language/move-lang/src/shared/mod.rs

```diff
@@ -72,19 +72,19 @@ impl AsRef<[u8]> for Address {
 
 impl fmt::Display for Address {
     fn fmt(&self, f: &mut fmt::Formatter) -> std::fmt::Result {
-        write!(f, "0x{:#x}", self)
+        write!(f, "0x{:#X}", self)
     }
 }
 
 impl fmt::Debug for Address {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "0x{:#x}", self)
+        write!(f, "0x{:#X}", self)
     }
 }
 
-impl fmt::LowerHex for Address {
+impl fmt::UpperHex for Address {
```

Comment:
Does implementing `UpperHex` give you `LowerHex` for free? And if not, can we keep the `LowerHex` impl?

Reply:
I'll keep both, yeah